application-xpoll-ui/src/main/resources/XPoll/XPollClass.xml

```diff
@@ -117,22 +128,34 @@
       <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
     </status>
     <type>
+      <cache>0</cache>
       <customDisplay/>
-      <defaultValue>0</defaultValue>
+      <defaultValue/>
       <disabled>0</disabled>
-      <displayFormType>select</displayFormType>
-      <displayType>1/0</displayType>
+      <displayType>select</displayType>
+      <freeText/>
+      <hint/>
+      <largeStorage>0</largeStorage>
+      <multiSelect>0</multiSelect>
       <name>type</name>
-      <number>6</number>
+      <number>3</number>
+      <picker>0</picker>
       <prettyName>Type</prettyName>
+      <relationalStorage>0</relationalStorage>
+      <separator> </separator>
+      <separators> ,|</separators>
+      <size>1</size>
+      <sort>none</sort>
       <unmodifiable>0</unmodifiable>
       <validationMessage/>
       <validationRegExp/>
-      <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+      <values>0=Allow multiple options|1=Can select a single option|2=Allow multiple options in order</values>
```

Comment:
The value should be the role hint of the ``PollResultsCalculator`` implementation.